src/modules/server.ts

```diff
@@ -0,0 +1,70 @@
+import Koa from 'koa'
+import Router from 'koa-router'
+import getPort from 'get-port'
+import asyncRetry from 'async-retry'
+import hello from './handlers/hello'
+// eslint-disable-next-line no-unused-vars
+import { Server } from 'http'
+
+class WebServer {
+  private app: Koa
+  private router: Router
+  private port: number
+  private server: Server
+
+  constructor () {
+    this.app = new Koa()
+    this.router = new Router()
+  }
+
+  public async getPort () {
+    return this.port
+  }
+
+  public async getApp () {
```

Comment:
What is the difference between public and private methods? And why this one is a public method?

Reply:
A public method can be accessed by any other object whereas a private method can not. This is a public method because it's needed in the test file, where a new server object is created.